Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -227,7 +264,10 @@ private async void LoginWithExistingAccount()
                      */
                     if (useWalletConnect)
                     {
-                        services.UseWalletConnectSigner(BuildWalletConnectConfig()).UseWalletConnectTransactionExecutor();
+                        services
+                            .UseWalletConnectProvider(BuildWalletConnectConfig())
```

Comment:
I believe it actually makes more sense to keep the uniformity, `JsonRPCSigner - WalletConnectSigner`, JsonRPCTransactionExecutor - `WalletConnectTransactionExecutor`, `JsonRPCProvider : IRPCProvider - WalletConnectProvider`

Reply:
I've got an idea. We can keep the WalletConnectProvider name, make it internal and register it by default whenever we call either UseWCSigner or UseWCTxExecutor.
Benefits: 
1) User won't need to know what's WalletConnectProvider.
2) We keep the name.
3) No UseWalletConnectProvider call as it gets registered automatically.

The only issue is configuration, but this can be done by a separate ConfigureWalletConnect(WalletConnectConfig) call.

What do you think?

Reply:
if the confusion is purely naming we can rename UseWCProvider to ConfigureWalletConnect, however I still don't see any reason for confusion and reason for a rename, we're setting up JsonRPC the same way

Reply:
Confusion is cause by the fact the we have a core component (slot) called 'Provider'. WalletConnectProvider is not 'Provider', however the name suggests the opposite.

Reply:
method name: UseWalletConnect(config)
class name: WalletConnectCustomProvider

Reply:
okay I'll make the renames

Reply:
done